### ISSUE TITLE
Add the ability to test HDR and wide color rendering

### ISSUE THREAD

**Issue by ccameron-chromium** [web-platform-tests/wpt#44320]
At present, all pixel tests capture the content as sRGB. This limits our ability to test wider color gamut and HDR content. This bug is to track this issue.

**Comment by svgeesus** [comment]
I agree. There is a need to run tests on an sRGB screen, a wide-gamut screen, and a screen that is both wide-gamut and HDR.

Currently for example the macOS runs all have sRGB screens and yet most current Apple hardware ships with a P3 screen.

**Comment by Minju-kim-igalia** [comment]
Cross-linking [Chromium issue 483413433](https://issues.chromium.org/issues/483413433) here, as I’m taking an initial look at what automated WPT coverage for JPEG XL HDR and wide-gamut color might require.

This also seems directly relevant to the Interop 2026 JPEG XL roadmap, which calls out HDR / wide-gamut testing explicitly.

I’ve opened a companion discussion in interop-jpegxl to scope the JPEG XL-specific side of this, including reference-output questions, the current Adobe RGB / Display P3 / Rec.2020 coverage, and initial HDR targets such as PQ / HLG while keeping gain maps in scope.
